ui/pages/Project/constants.js

```diff
@@ -160,89 +160,91 @@ const ASSIGNED_TO_ME_FILTER = {
   createFilter: (individualsByGuid, user) => family => familyIsAssignedToMe(family, user),
 }
 
-export const FAMILY_FILTER_OPTIONS = [
```

Comment:
the filter options themselves did not change, but now they are a dictionary grouped by category instead of having a `category` field